app/views/layouts/application.html.erb

```diff
@@ -25,12 +25,21 @@
   <%= render "govuk_publishing_components/components/layout_header", {
     environment: GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment,
     navigation_items: [
+      { text: "Switch app", href: Plek.new.external_url_for("signon") },
       { text: current_user.name, href: Plek.new.external_url_for("signon") },
       { text: "Log out", href: gds_sign_out_path }
     ]
   }%>
 
   <div class="govuk-width-container">
+    <%= render "govuk_publishing_components/components/phase_banner", {
+      app_name: "Content publisher",
+      phase: "beta",
+      message: capture do
+        render "layouts/phase_banner_elements"
```

Comment:
I'm not sure we need a partial for this - we could capture it above - but if we do `layouts` isn't probably the best place. I would check with @benthorner to see where it fits in the project structure.

Reply:
@benthorner - where would you put something like this, I put it in here as it is a generic element used across the whole app.

Reply:
I think I'm more concerned about the footer part, which has made this file long. I think it's fine to make a few partials here, as this file is becoming unreadable for someone trying to get an idea of its structure. And layouts/_partial.html.erb is the place I'd look first to find them - you might need to specify the path explicitly when you render them, though.